app/src/main/java/com/github/mobile/ui/ViewPager.java

```diff
@@ -61,7 +59,7 @@ public boolean setItem(final int item) {
     /**
      * Set current item, invoke the listener if changes, and return whether the
      * item changed
-     * <p>
+     * <p/>
```

Comment:
According to [Oracle's Javadoc tutorial](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html) <p> is correct.